src/game/game.py

```diff
@@ -5,7 +5,50 @@
 class Game(polymodel.PolyModel):
     """Represents a single instance of a Game"""
     date = ndb.DateTimeProperty(auto_now_add=True, required=True)
-    player_keys = ndb.KeyProperty(kind="User", repeated=True, required=True)
+    player_keys = ndb.KeyProperty(kind="User", repeated=True)
```

Comment:
We have 3 instances of `user` stored on this entity. I'm wondering if there's any way we could reduce that.